assets/queries/cloudFormation/s3_bucket_with_all_permissions/query.rego

```diff
@@ -1,40 +1,24 @@
 package Cx
 
-CxPolicy[result] {
-	resource := input.document[i].Resources[name]
-	resource.Type == "AWS::S3::BucketPolicy"
-	statement := resource.Properties.PolicyDocument.Statement[j]
-	statement.Effect == "Allow"
-	statement.Resource == "*"
-	check_action(statement.Action[k])
-
-	result := {
-		"documentId": input.document[i].id,
-		"searchKey": sprintf("Resources.%s.Properties.PolicyDocument.Statement", [name]),
-		"issueType": "IncorrectValue",
-		"keyExpectedValue": sprintf("Resources.%s.Properties.PolicyDocument.Statement does not allow all actions from all principals", [name]),
-		"keyActualValue": sprintf("Resources.%s.Properties.PolicyDocument.Statement allows all actions from all principals", [name]),
-	}
-}
```

Comment:
Why this rule was removed?

Reply:
Because the difference between both rules was the verification of Action as a string or an array. This "new rule" verifies both cases